src/Registry/RepositoryRegistry.php

```diff
@@ -73,7 +73,11 @@ public function all()
      */
     public function get($repositoryName = null)
     {
-        if (!isset($this->serviceMap[$repositoryName ?: $defaultRepositoryName])) {
+        if (null === $repositoryName) {
+            $repositoryName = $this->defaultRepositoryName;
```

Comment:
I pushed a commit to master today reintroducing the concept of a default repository. The `defaultRepositoryName` is passed to the constructor (and can be configured in the configuration).